elements/pfe-accordion/demo/index.html

```diff
@@ -84,7 +84,7 @@
 
       .custom-styles pfe-accordion {
         --pfe-accordion--accent: #fff;
-        --pfe-theme--color--ui-base: #444;
+        --pfe-theme--color--ui-accent: #444;
```

Comment:
Dark gray doesn't really seem like an accent to me.  When I think of a design accent, I imagine a small pop of some color. cc @coreyvickery

Reply:
Updated to a color with some hue.